src/model/model.js

```diff
@@ -0,0 +1,91 @@
+import { EventEmitter } from "events";
+
+class Model extends EventEmitter {
+    constructor() {
+        super();
+        this.state = {
+            questions: [{
+              id:1,
+                author:"Ana",
+                title:"SQL problem",
+                text:"Connection problem",
+                creationDate:"11/02/2019",
+                tag:"SQL"
+            }, {
+                id:2,
+                author:"Mara",
+                title:"Linux problem",
+                text:"Random error",
+                creationDate:"18/02/2019",
+                tag:"LINUX"
+            }],
+            newQuestion: {
+               id:"",
+                author:"",
+                title:"",
+                text:"",
+                creationDate:"",
+                tag:""
+            },
+            searchQuestions:{},
+            toSearch:"",
+            currentId:2
+            
+        };
+    }
+
+    addQuestion(author,title,text,creationDate,tag) {
+        this.state = {
+            ...this.state,
+            currentId:++this.state.currentId,
```

Comment:
Very close. By doing `++` you are actually changing the previous state (as `++this.state.currentId` means `this.state.currentId = this.state.currentId + 1`). You should just do ` + 1`.